### ISSUE TITLE
Add the search capability in plugin browser

### ISSUE THREAD

**Issue by ykcab** [matryer/xbar#633]
It will be nice to have the ability to search for a certain `x` plugin in the plug-in browser. A future rich word/text search would be sublime! 😊

**Comment by DaniruKun** [comment]
@matryer any update on this? Even a simple pattern search would already be leaps and bounds better than nothing.

**Comment by magicus** [comment]
I agree. I was mightily surprised to see that there was no way to search. I was looking for a plugin for Toodledo, and I had no idea what category an author would put it in, nor if the name would start with "T". So it took some time looking through the categories I thought was possible. (Only to find there were no such plugin :-()

Any kind of search would be an improvement.

**Comment by m0rg5** [comment]
+1 a year later!

**Comment by lderugeriis** [comment]
I second this
